neurom/core/population.py

```diff
@@ -74,19 +74,22 @@ def __init__(self, files, name='Population', ignored_exceptions=(), cache=False,
 
         self._files = _resolve_if_morphology_paths(files)
 
+        self._process_subtrees = process_subtrees
+
         if cache:
-            self._files = [self._load_file(f) for f in self._files if f is not None]
+            self.reset_cache()
 
-        self._process_subtrees = process_subtrees
+    def reset_cache(self):
```

Comment:
As it is currently implemented I don't think so. My initial thinking was that it could be used to reload the files (for example if the user know that the files changed during runtime and he wants to reload them) but once the morphs are loaded they are not reloaded, just copied. So we could force to reload the files in this function but I think it's indeed simpler to just not make it public for now.